src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -215,7 +215,7 @@ public void cellUpdate( final MENetworkCellArrayUpdate ev )
 	{
 		this.storageNetworks.clear();
 
-		final List<ICellProvider> ll = new LinkedList<ICellProvider>();
+		final List<ICellProvider> ll = new ArrayList<ICellProvider>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2293)